lib/src/model/broadcast/broadcast_providers.dart

```diff
@@ -54,6 +55,50 @@ Future<BroadcastTournament> broadcastTournament(
   );
 }
 
+enum BroadcastPlayersSortingTypes { player, elo, score }
+
+@riverpod
+class BroadcastPlayers extends _$BroadcastPlayers {
+  @override
+  Future<IList<BroadcastPlayerExtended>> build(
+    BroadcastTournamentId tournamentId,
+  ) async {
+    final players = ref.withClient(
+      (client) => BroadcastRepository(client).getPlayers(tournamentId),
+    );
+
+    return players;
+  }
+
+  void sort(BroadcastPlayersSortingTypes sortingType, [bool reverse = false]) {
```

Comment:
Looks like this function is not used since the sorting is done in the widget state.

Reply:
Oops I forgot to remove it when moving the sort in the widget